src/Microsoft.HttpRepl.Tests/OpenApi/EndpointMetadataReaderTests.cs

```diff
@@ -0,0 +1,495 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.HttpRepl.OpenApi;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.HttpRepl.Tests.OpenApi
+{
+    public class EndpointMetadataReaderTests
```

Comment:
This feels like a suite of tests covering `OpenApiV3EndpointMetadataReader`, `SwaggerV2EndpointMetadataReader`, and `SwaggerV1EndpointMetadataReader`, but there's no validation of the intricacies of the responses (e.g. parameter details).  Should these be split out into tests for the respective metadata readers?

Reply:
Similarly, I don't see testing here around registering an additional reader.

Reply:
Split the tests here:
https://github.com/aspnet/HttpRepl/commit/e95a6c7efc23b60ce7efccda7c49fbf286d3e68c